src/components/flat-table/flat-table-sizes.stories.mdx

```diff
@@ -0,0 +1,59 @@
+import { Meta, Props, Preview, Story } from '@storybook/addon-docs/blocks';
+import Box from "../box"
+import { 
+  FlatTable, 
+  FlatTableHead, 
+  FlatTableBody, 
+  FlatTableRow, 
+  FlatTableHeader,
+  FlatTableCell,
+} from '.';
+
+<Meta title="Design System/Flat Table/Sizes" parameters={{ info: { disable: true }}} />
+
+## Table Cell Sizes
+
+<Preview>
+  <Story name="compact">
+    <div>
+      {["compact", "small", "medium", "large"].map(size => <Box mb="3">
+        <FlatTable size={size}>
+          <FlatTableHead>
+            <FlatTableRow>
+              <FlatTableHeader>Name</FlatTableHeader>
+              <FlatTableHeader>Location</FlatTableHeader>
+              <FlatTableHeader>Relationship Status</FlatTableHeader>
+              <FlatTableHeader>Dependents</FlatTableHeader>
```

Comment:
Can we add or change a row to show an input box. We can then show that the input box changes size to correspond to the DLS

Reply:
input boxes should be sized separately as they have their own `size` prop